src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,257 @@
++ String extId 
+
+#Census Entities
+####LocationHierarchy
+(child of `AuditableExtIdEntity`)
+#####What is it?
+`LocationHierarchy` is a node in the tree that represents the study area. For example, at the top of a tree would be the The United States and it would have a child node for each of the 50 states. Each state node could have a child node for its cities and each city could have a child node for its districts and so on. `LocationHierarchy` extends `AuditableExtIdEntity` meaning it is identified by uuid as well as extId. 
+#####Dependencies
++ FieldWorker
++ LocationHierarchy
++ LocationHierarchyLevel
+
+######Required Fields:
++ String name
++ String extId
++ LocationHierarchy parent
++ LocationHierarchyLevel level
+
+___
+####LocationHierarchyLevel
+(child of `AuditableEntity`)
+#####What is it?
+The levels of the `LocationHierarchy` tree are defined separately as `LocationHierarchyLevel`s and referenced by each instance of `LocationHierarchy`. From the previous example, the list of `LocationHierarchyLevel`s would be Country, State, City, District.
+######Required Fields:
++ int keyIdentifier
++ String name
+
+___
+####Location
+(child of `AuditableExtIdEntity`)
+#####What is it?
+`Location` is the leaf node on the `LocationHierarchy` tree, it is the smallest geographical unit within a study area. From the previous example for `LocationHierarchy`, each building within a district could be modeled with `Location`. `Individual`s are residents at `Locations`.
+######Dependencies
++ FieldWorker
++ LocationHierarchy
++ Domain Constraints
+
+######Required Fields:
++ String name
++ String extId
++ LocationHierarchy parent
+
+___
+####Individual
+(child of `AuditableExtIdEntity`)
+#####What is it?
+An `Individual` (person) within the study area that resides at a `Location`, is a member of a `SocialGroup`, and has a relationship to the head of a household.
+######Dependencies
++ FieldWorker
+
+######Required Fields:
++ String extId
++ String gender
++ Date dateOfBirth
++ String firstName
+
+___
+####SocialGroup
+(child of `AuditableExtIdEntity`)
+#####What is it?
+A `SocialGroup` is any cultural or societal group or collective that `Individual`s are a member of, e.g. a church group, a household/family, a bowling team.
+######Dependencies
++ FieldWorker
+
+######Required Fields:
++ String groupName
++ String groupType
++ String extId
+
+___
+###Membership
+(child of `AuditableCollectedEntity`)
+####What is it?
+A `Membership` models the relationship between an `Individual` and a `SocialGroup`.
+######Dependencies
++ FieldWorker
++ Individual
++ SocialGroup
+
+######Required Fields:
++ Date startDate
++ String startType
++ Individual individual
++ SocialGroup socialGroup
+
+___
+####Relationship
+(child of `AuditableCollectedEntity`)
+#####What is it?
+A `Relationship` models the relationship between two `Individual`s. Specifically it models the relationship between an `Individual` and the head of household that the `Individual` is a part of.

```

Comment:
Not all Relationships have to be part of a household.  So I think you should leave out the relationship to head of household here.

Or, make it part of a separate paragraph and make it clear that that is just one example.